src/pal/src/include/pal/stackstring.hpp

```diff
@@ -144,6 +161,24 @@ class StackString
         return;
     }
     
+    //count Should not include the terminating null
+    BOOL Append(const T * buffer, SIZE_T count)
```

Comment:
I will keep the modifications but introduce a new Function called Allocate to pre-allocate memory if we know beforehand the length. sounds good ?